assets/stylesheets/sass/main/main.scss

```diff
@@ -416,19 +458,23 @@ figure {
   }
 
   .video-content {
-    h1{
+
+    h1 {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Selector should have depth of applicability no greater than 2, but was 3